pubsub/cloud-client/subscriptions.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to perform basic operations on
+subscriptions with the Google Cloud Pub/Sub API.
+
+For more information, see the README.md under /pubsub and the documentation at
+https://cloud.google.com/pubsub/docs.
+"""
+
+import argparse
+import os
+
+from google.cloud import pubsub
+
+
+pubsub_client = pubsub.Client()
```

Comment:
This is no bueno:

* It's a global variable (not a constant)
* There should not be side-effects for top-level statements.
* This is not easily testable.